doc/operation-and-maintenance/System-Metrics-Privacy-Policy.md

```diff
@@ -0,0 +1,89 @@
+Moreover, user can configure Tracking ID for their own Google Analytics account and have a view on their MongooseIM status in that dashboard.
+For more information on how to set up the Tracking ID, please see [How to configure additional and private Tracking ID in Google Analytics.](#How-to-configure-additional-and-private-Tracking-ID-in-Google-Analytics?).
+
+Full list of information that is being gathered can be seen below:
+
+* MongooseIM node uptime.
+* MongooseIM version.
+* Number of nodes that are part of the MongooseIM cluster.
+* Generic modules that are part of the opensource project and are in use. Some modules report what database they use as a backend, e.g. [Sample report](#How-the-information-is-being-used?).
+* Number of custom modules - without disclosing any details, we are just curious to see if there are any.
+* Number of connected external XMPP components.
+* List of configured REST APIs that are part of the opensource project.
+* XMPP transport mechanisms like:TCP/TLS, WebSockets or BOSH.
+* Geographical Data - Google Analytics is providing several geographical dimensions, such as City, Country, Continent. These values are derived from the IP address, the data was sent from. See [About Geographical Data](https://support.google.com/analytics/answer/6160484?hl=en) for more details.
+
+# How the information is being used?
+
+
+Information collected is automatically anonymised before it is being processed further.
+Each MongooseIM is generating random Client ID that is being attached to the reports.
+The data collected has only statistical relevance and aims to help us understand the needs of our users.
+Knowing how our product is used will point us into the direction on how should we focus further efforts while developing the product.
+
+# How a report looks like?
+Sample report showing metrics for the mod_vcard backends from Google Analytics can be found below.
+![System metrics sample report][system_metrics_report]
+
+Based on such report we can see the frequency of different backends being used with mod_vcard.
+
+# How often the metrics are reported?
+Metrics are reported first shortly after the system startup and later at regular intervals.
+These timers are configuration parameters `initial_report` and `periodic_report`.
+The default values are 5 minutes for the initial report and 3 hours for the periodic one.
+These reporting intervals can be changed depending on the configuration parameters.
+
+# How to configure this service?
+This functionality is provided as a "service".
+For more details regarding service configuration, please see [Services](/advanced-configuration/Services) section.
+
+# How to configure additional and private Tracking ID in Google Analytics?
+The data that is gathered and forwarded to Google Analytics.
+User can add Google Analytics Tracking ID in the MongooseIM configuration and see all incoming events that are related their own system metrics being reported.
```

Comment:
```suggestion
The user can add custom Google Analytics Tracking ID in the MongooseIM configuration and see all incoming events that are related to their own system metrics being reported.
```
I'm starting to notice a pattern where you use `User` as a proper name and hence devoid of the need for an article, but I'm not sure I'm convinced 🙁

Reply:
:D , we can go with an article